protocol/listpartitionreassignments/listpartitionreassignments.go

```diff
@@ -0,0 +1,58 @@
+package listpartitionreassignments
+
+import "github.com/segmentio/kafka-go/protocol"
+
+func init() {
+	protocol.Register(&Request{}, &Response{})
+}
+
+// Detailed API definition: https://kafka.apache.org/protocol#The_Messages_ListPartitionReassignments.
+
+type Request struct {
+	// We need at least one tagged field to indicate that this is a "flexible" message
+	// type.
+	_ struct{} `kafka:"min=v0,max=v0,tag"`
+
+	TimeoutMs int32          `kafka:"min=v0,max=v0"`
+	Topics    []RequestTopic `kafka:"min=v0,max=v0,nullable"`
+}
+
+type RequestTopic struct {
+	Name             string  `kafka:"min=v0,max=v0"`
```

Comment:
I think we need tags on these nested data structures in this file as well

Reply:
I must also add it on all response structures, right?

```
type ResponseTopic struct {
	Name       string              `kafka:"min=v0,max=v0"`
	Partitions []ResponsePartition `kafka:"min=v0,max=v0"`
}

type ResponsePartition struct {
	PartitionIndex   int32   `kafka:"min=v0,max=v0"`
	Replicas         []int32 `kafka:"min=v0,max=v0"`
	AddingReplicas   []int32 `kafka:"min=v0,max=v0"`
	RemovingReplicas []int32 `kafka:"min=v0,max=v0"`
}

```

Reply:
Yes, those as well as `RequestTopic`